release.json

```diff
@@ -2,7 +2,7 @@
 	"nightly": {
 		"INTEGRATIONS_CORE_VERSION": "master",
 		"OMNIBUS_SOFTWARE_VERSION": "master",
-		"OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
+		"OMNIBUS_RUBY_VERSION": "db/sign_with_file",
```

Comment:
needs to be switched back to `datadog-5.5.0` before merging

Reply:
We'll need to merge this first: https://github.com/DataDog/omnibus-ruby/